### PR TITLE
Lifting State and Props Handling

### PR DESCRIPTION
**Pull request by briHalterman**

### CODE REVIEW

File: src/AddTodoForm.jsx [hunk 1]
```diff
@@ -1,22 +1,36 @@
...
-function AddTodoForm(props) {
   function handleAddTodo(event) {
     event.preventDefault();
 
-    const todoTitle = event.target.title.value;
-
-    console.log(todoTitle);
+    const newTodo = {
+      title: todoTitle,
+      id: Date.now(),
```
Comment (royemosby) [inline src/AddTodoForm.jsx:16]: 👍 Ideally, we would let the backend create the ids for us.

File: src/TodoListItem.jsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
-function TodoListItem(props) {
-  return <li>{props.todo.title}</li>;
+function TodoListItem({ todo }) {
```
Comment (royemosby) [inline src/TodoListItem.jsx:3]: 👍

**Review by royemosby — APPROVED** [review]
Looks good to merge 🚀

File: src/TodoListItem.jsx (hunk 2 above)
Comment (briHalterman) [reply src/TodoListItem.jsx:3]: Thanks Roy!

**Pull request merged**